### PR TITLE
Update ingress multiple host

### PR DESCRIPTION
**Pull request by jsenon**
Add warning. #8073

### CODE REVIEW

**Comment by frankbu** [comment]
What issue is this addressing? #8073 doesn't exist

File: content/docs/tasks/traffic-management/secure-ingress/index.md [hunk 1]
```diff
@@ -339,6 +339,9 @@ In this subsection, perform the same steps as in the [Generate client and server
 
 ### Configure traffic for the `bookinfo.com` host
 
+> {{< warning_icon >}} port name must be distinct for HTTPS.
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/secure-ingress/index.md]: This warning doesn't seem appropriate here. Which port name is it referring to? I don't see an instruction that requires the user to name a port.

**Comment by jsenon** [comment]
This one https://github.com/istio/istio/issues/8073

File: content/docs/tasks/traffic-management/secure-ingress/index.md (hunk 1 above)
Comment (jsenon) [reply content/docs/tasks/traffic-management/secure-ingress/index.md]: In the gateway definition yaml file. Don’t seen other place to add this comment. Let me know if you have any idea

Comment (frankbu) [reply content/docs/tasks/traffic-management/secure-ingress/index.md]: I think the place to document this issue is [here](https://istio.io/help/ops/traffic-management/deploy-guidelines/).

Comment (frankbu) [reply content/docs/tasks/traffic-management/secure-ingress/index.md]: Maybe also mention the restriction in the description of the `port` property of ref docs?
https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Server

Comment (jsenon) [reply content/docs/tasks/traffic-management/secure-ingress/index.md]: I can but for my understanding "Configure a TLS ingress gateway for multiple hosts" and add a warning on this section is the correct one. This issue only apply when you deploy multiple hosts with same port name for https on the gateway.

Comment (frankbu) [reply content/docs/tasks/traffic-management/secure-ingress/index.md]: @rshriram told me we may be able to remove this restriction from the code, so we won't need anything in the doc. I reopend the istio/istio issue.

Comment (jsenon) [reply content/docs/tasks/traffic-management/secure-ingress/index.md]: Ok, let me know if you want to close the 2 PR has requested

**Comment by geeknoid** [comment]
Should we close this PR or should it be merged?

**Comment by frankbu** [comment]
Closing, this is already fixed.
https://preliminary.istio.io/help/ops/traffic-management/deploy-guidelines/#configuring-multiple-tls-hosts-in-a-gateway

**Pull request closed without merging**